modules/nextflow/src/main/groovy/nextflow/processor/TaskId.groovy

```diff
@@ -38,6 +38,8 @@ class TaskId extends Number implements Comparable, Serializable, Cloneable {
 
     private final int value
 
+    int getValue() { value }
```

Comment:
There is already `intValue()` for this, see below

Reply:
removed in last commits